File: CS/CellStyleExample/MainViewModel.cs
using DevExpress.Mvvm;
using System;
using System.Collections.ObjectModel;
using System.Linq;

namespace CellStyleExample {
    public class MainViewModel : ViewModelBase {


        protected ObservableCollection<Order> _Items;
        public ObservableCollection<Order> Items
        {
            get
            {
                if(this._Items == null)
                {
                    Random r = new Random();
                    this._Items = new ObservableCollection<Order>(Enumerable.Range(0, 10).Select(c => new Order() {
                        OrderId = 1000 + c,
                        CustomerName = "Customer #" + c,
                        Price = Math.Round(r.NextDouble() * 100, 2),
                        Quantity = r.Next(5) + 1
                    }));
                }

                return this._Items;
            }
        }

    }
}

File: VB/CellStyleExample/MainViewModel.vb
Imports DevExpress.Mvvm
Imports System
Imports System.Collections.ObjectModel
Imports System.Linq

Namespace CellStyleExample
    Public Class MainViewModel
        Inherits ViewModelBase


        Protected _Items As ObservableCollection(Of Order)
        Public ReadOnly Property Items() As ObservableCollection(Of Order)
            Get
                If Me._Items Is Nothing Then
                    Dim r As New Random()
                    Me._Items = New ObservableCollection(Of Order)(Enumerable.Range(0, 10).Select(Function(c) New Order() With { _
                        .OrderId = 1000 + c, _
                        .CustomerName = "Customer #" & c, _
                        .Price = Math.Round(r.NextDouble() * 100, 2), _
                        .Quantity = r.Next(5) + 1 _
                    }))
                End If

                Return Me._Items
            End Get
        End Property

    End Class
End Namespace
